src/index.ts

```diff
@@ -1,8 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as vscodeUri from 'vscode-uri';
-import { NotebookConverter } from './notebookConverter';
+import { NotebookConverterImpl, NotebookConverter } from './notebookConverter';
+import { getConcatDocumentRoot } from './notebookConcatDocument';
+import { isNotebookCell } from './common/utils';
+import { RefreshNotebookEvent } from './types';
 
-export function createConverter(getNotebookHeader: (uri: vscodeUri.URI) => string, platformGetter: () => string): NotebookConverter {
-    return new NotebookConverter(getNotebookHeader, platformGetter);
+export { isNotebookCell, getConcatDocumentRoot, RefreshNotebookEvent, NotebookConverter };
+
+export function createConverter(
```

Comment:
any reason we have this function rather than exporting NotebookConverter class?

Reply:
In case we want to change the implementation details later. We have this factory method to abstract out the constructor so that users of the npm module don't need to change their calls.